nix/README.md

```diff
@@ -131,8 +126,67 @@ nixpkgs with some overlays applied (see
 provided to the final Mina derivation. See [./ocaml.nix](./ocaml.nix)
 for more details.
 
-## Troubleshooting
+### Utility
```

Comment:
I think this whole section should be moved up a bit, and renamed

```suggestion
# direnv
```

Reply:
@balsoft thank you, 
like put it under the `Use It` section and rename to `Dev experience`?